Resources/Textures/_CM14/Structures/Walls/almayer_black.rsi/meta.json

```
@@ -0,0 +1,52 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/f74929c306a5d2ec5832abcdc695aea1b0786664/icons/turf/walls/almayer.dm",
```

Missing an `i` at the end of the link, otherwise it's a broken link. .dm files are code files, .dmi are image files

```suggestion
  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/f74929c306a5d2ec5832abcdc695aea1b0786664/icons/turf/walls/almayer.dmi",
```